### PR TITLE
RadioGroup: add Vue, React and Angular code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: concepts/05 Widgets/RadioGroup/00 Overview.md [hunk 1]
```diff
@@ -17,10 +20,87 @@ The following code adds a simple **RadioGroup** to your page. Here, the [value](
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-radio-group
+        [data-source]="dataSource"
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/RadioGroup/00 Overview.md]: In Angular, attributes use camelCase notation.

File: concepts/05 Widgets/RadioGroup/00 Overview.md [hunk 2]
```diff
@@ -36,8 +116,103 @@ If your data is an array of objects, bind it to the **RadioGroup** using the [di
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-radio-group
+        [data-source]="dataItems"
+        [value]="dataItems[1]"
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/RadioGroup/00 Overview.md]: Doesn't look like a good idea. It is better to bind to a component property in Angular and Vue and use the state in React.

File: concepts/05 Widgets/RadioGroup/10 Customize Item Appearance.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 For a minor customization of **RadioGroup** items, you can define [specific fields](/api-reference/10%20UI%20Widgets/DataExpressionMixin/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxRadioGroup/Configuration/items/') in item data objects. For example, the following code generates three radio buttons: the first is disabled, the second is not customized, the third is hidden.
 
+---
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/RadioGroup/10 Customize Item Appearance.md:3]: Same note as [here](https://github.com/DevExpress/devextreme-documentation/pull/658#discussion_r409374999)

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/RadioGroup/00 Overview.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/RadioGroup/00 Overview.md]: Oh, sorry.. fixed

File: concepts/05 Widgets/RadioGroup/00 Overview.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/RadioGroup/00 Overview.md]: fixed

File: concepts/05 Widgets/RadioGroup/10 Customize Item Appearance.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/RadioGroup/10 Customize Item Appearance.md:3]: fixed

File: concepts/05 Widgets/RadioGroup/05 Handle the Value Change Event.md [hunk 4]
```diff
@@ -11,18 +14,131 @@ To process a new **RadioGroup** value, you need to handle the value change event
...
+                />
+            );
+        }
+    }
+
+    export default App;
+
+---
+
 If you are going to change event handlers at runtime, or if you need to attach several handlers to the value change event, subscribe to this event using the [on(eventName, eventHandler)](/api-reference/10%20UI%20Widgets/Component/3%20Methods/on(eventName_eventHandler).md '/Documentation/ApiReference/UI_Widgets/dxRadioGroup/Methods/#oneventName_eventHandler') method.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/RadioGroup/05 Handle the Value Change Event.md]:
```suggestion
If you are going to change event handlers at runtime, or if you need to attach several handlers to the value change event, subscribe to this event using the [on(eventName, eventHandler)](/api-reference/10%20UI%20Widgets/Component/3%20Methods/on(eventName_eventHandler).md '/Documentation/ApiReference/UI_Widgets/dxRadioGroup/Methods/#oneventName_eventHandler') method. This approach is more typical of jQuery.
```

**Review by RomanTsukanov — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**